config/storage.yml

```diff
@@ -1,10 +1,13 @@
-test:
-  service: Disk
-  root: <%= Rails.root.join("tmp/storage") %>
+cloudinary:
```

Comment:
Looks like I missed this in my Cloudinary install/config. Good catch and sorry for the trouble.